tokens/sandiegocoin/SAN.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "SAND",
+  "symbol": "SAND",
+  "contract": "SAND-eos-sandiegocoin",
```

Comment:
Please fill in with real eos contract

Reply:
ok done, sorry it was supposed to juyst be sandiegocoin ok changed